docs/001_develop/03_client-capabilities/006_charts/002_g2plot-chart/03_chart_html_api.mdx

```diff
@@ -0,0 +1,430 @@
+<td>
+
+```typescript
+<chart-datasource maxView="500">
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>orderBy</td>
+            <td><code>string</code></td>
+            <td>This option can be used to select a Data Server index (defined in the Data Server query), which is especially useful if you want the data to be sorted in a specific way.</td>
+<td>
+
+```typescript
+<chart-datasource orderBy="TRADE_DATETIME">
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>reverse</td>
+            <td><code>boolean</code></td>
+            <td>This option changes the Data Server index iteration.</td>
+<td>
+
+```typescript
+<chart-datasource reverse>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>rowId</td>
+            <td><code>string</code></td>
+            <td>Attribute to set an unique identifier for the row.</td>
+<td>
+
+```typescript
+<chart-datasource rowId="ROW_REF">
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>is-stack</td>
+            <td><code>boolean</code></td>
+            <td>If true, will enable stacked chart to effectively portray comparisons of total values across several categories</td>
+<td>
+
+```typescript
+<chart-datasource is-stack>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>is-group</td>
+            <td><code>boolean</code></td>
+            <td>If true, will enable grouped chart plotting numeric values for levels of two categorical variables instead of one.</td>
+<td>
+
+```typescript
+<chart-datasource is-group>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>reduced</td>
+            <td><code>boolean</code></td>
+            <td>Enabled by default, will not work if a custom charts-fields is used (requires [type, value])</td>
+<td>
+
+```typescript
+<chart-datasource reduced>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>with-timestamp-formatting</td>
+            <td><code>boolean</code></td>
+            <td>Enabled by default, will format UNIX timestamp fields to readable `DATE|DATETIME` values</td>
+<td>
+
+```typescript
+<chart-datasource with-timestamp-formatting>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>show-formatted-time</td>
+            <td><code>boolean</code></td>
+            <td>Disabled by default, will ignore the TIME portion in the Chart labels. Only works if `withTimestampFormatting` is true. Only affects `DATETIME` fields.</td>
+<td>
+
+```typescript
+<chart-datasource show-formatted-time>
+```
+
+</td>
+        </tr>
+    </tbody>
+</table>
+
+#### Properties
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Use</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>request</td>
+      <td><code>any</code></td>
+      <td>This optional parameter enables you to specify request fields, which can include wildcards..</td>
+<td>
+
+```typescript
+<chart-datasource :request=${(x) => x.datasourceRequest}>
```

Comment:
Given this is an any, can we have a concrete example of what `datasourceRequest` would be?